docs/examples/intro/local.md

```diff
@@ -115,32 +126,23 @@ We provide a conda yaml in out `local_folder` which tempo will use as the runtim
 
 
 ```python
```

Comment:
* Wrong location for conda.yaml
 * Also I changed to use file syntax for consistency and clarity
 * wrong version of mlserver

```
%%writetemplate artifacts/classifier/conda.yaml
name: tempo
channels:
  - defaults
dependencies:
  - python=3.7
  - pip:
    - mlops-tempo @ file://{TEMPO_DIR}
    - mlserver==0.3.1.dev7
 ```

Also not need to remove this file from github and add to .gitignore as its generated.